common/static/common/js/discussion/utils.js

```diff
@@ -386,26 +410,28 @@
                         return $3;
                     });
                 } else {
-                    processedText += text;
+                    processedText += safeHtml;
                     break;
                 }
             }
             /* jshint +W083 */
-            text = processedText;
-            text = text.replace(new RegExp(ESCAPED_DOLLAR, 'g'), '\\$');
-            text = text.replace(/\\\\\\\\/g, ESCAPED_BACKSLASH);
-            text = text.replace(/\\begin\{([a-z]*\*?)\}([\s\S]*?)\\end\{\1\}/img, function($0, $1, $2) {
+            safeHtml = processedText;
+            safeHtml = safeHtml.replace(new RegExp(ESCAPED_DOLLAR, 'g'), '\\$');
+            safeHtml = safeHtml.replace(/\\\\\\\\/g, ESCAPED_BACKSLASH);
+            safeHtml = safeHtml.replace(/\\begin\{([a-z]*\*?)\}([\s\S]*?)\\end\{\1\}/img, function($0, $1, $2) {
                 return processor(("\\begin{" + $1 + "}") + $2 + ("\\end{" + $1 + "}"));
             });
-            text = text.replace(new RegExp(ESCAPED_BACKSLASH, 'g'), '\\\\\\\\');
-            $div = $("<div>").html(text);
+            safeHtml = safeHtml.replace(new RegExp(ESCAPED_BACKSLASH, 'g'), '\\\\\\\\');
+            // safe-lint: disable=javascript-jquery-html
+            $div = $("<div>").html(safeHtml);
```

Comment:
See other comment about ensuring this is safe:
https://github.com/edx/edx-platform/pull/12914/files#r69576197